app/src/main/java/com/example/cactusnotes/RVAdapter.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.cactusnotes
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import android.widget.Toast
+import androidx.cardview.widget.CardView
+import androidx.recyclerview.widget.RecyclerView
+
+class RVAdapter(private val mContext: Context, private val notesList: List<NotesDate>) :
+    RecyclerView.Adapter<RVAdapter.CardViveDesignObjectsTaker>() {
+    inner class CardViveDesignObjectsTaker(view: View) : RecyclerView.ViewHolder(view) {
+
+        var noteCardview: CardView
+        var titleTextview: TextView
+        var paragraphTextview: TextView
+
+        init {
+            noteCardview = view.findViewById(R.id.noteCardview)
+            titleTextview = view.findViewById(R.id.titleTextview)
+            paragraphTextview = view.findViewById(R.id.paragraphTextview)
```

Comment:
You could inline these assignments:

val noteCardview: CardView = view.findViewById(R.id.noteCardview)

that way you can use val instead of var.